integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CommonLoadUtils.scala

```diff
@@ -982,9 +985,29 @@ object CommonLoadUtils {
           persistedRDD = persistedRDDLocal
           transformedPlan
         } else {
+          val partitionBasedOnLocality = CarbonProperties.getInstance()
```

Comment:
can move this to above and reuse in the else case also

Reply:
Done